src/main/java/org/labs/lab1/LIC.java

```diff
@@ -30,7 +136,7 @@ public static double calculateAngle(Point2D.Double a, Point2D.Double b, Point2D.
 
     /**
      * Finds a point triplet from a given respective point distance between them
-     * 
```

Comment:
Annoying that the formatting differs. Probably my fault for switching to VSCode. Ill think of a solution